assets/js/modules/pagespeed-insights/dashboard/dashboard-widget-speed.js

```diff
@@ -25,12 +25,17 @@ import { __ } from '@wordpress/i18n';
 /**
  * Internal dependencies
  */
-import Layout from '../../../components/layout/layout';
+import Data from 'googlesitekit-data';
 import DashboardModuleHeader from '../../../components/dashboard/dashboard-module-header';
 import DashboardPageSpeed from '../components/DashboardPageSpeed';
+import { STORE_NAME } from '../../../googlesitekit/datastore/site/constants';
+const { useSelect } = Data;
 
 function DashboardSpeed() {
-	const description = global.googlesitekit.permaLink ? __( 'How fast this page is.', 'google-site-kit' ) : __( 'How fast your home page is.', 'google-site-kit' );
+	const currentEntityURL = useSelect( ( select ) => select( STORE_NAME ).getCurrentEntityURL() );
+	const description = currentEntityURL
```

Comment:
This technically works, however it may be safer to use both `getCurrentReferenceURL` and `getCurrentEntityURL` and then make this condition `currentReferenceURL === currentEntityURL`. Currently, this code only works based on the internals of `getCurrentReferenceURL`, so by checking both we would decouple that.